gradle.properties

```diff
@@ -14,19 +14,19 @@
 android.enableJetifier=true
 
 org.gradle.jvmargs=-Xmx2048M -Dkotlin.daemon.jvm.options\="-Xmx2048M"
-org.gradle.unsafe.configuration-cache=true
 
 gradlePluginVersion=7.3.1
 googleServicesVersion=4.3.12
-kotlinVersion=1.8.21
+kotlinVersion=1.9.24
```

Comment:
We don't want 1.9.25 or the newly stable 2.0.0?
https://kotlinlang.org/docs/releases.html#release-details

Reply:
The Kotlin version is tied by the Compose Compiler: https://developer.android.com/jetpack/androidx/releases/compose-kotlin, the latest non-2.0 version compose compiler supports is 1.9.24

To update Kotlin to 2.0 some other things need to be done, I would very much prefer doing those things later

Reply:
Ah, right - no worries - 1.9.24 it is, then! =D